roles/kube_prometheus/files/prometheus_ingress.yml

```diff
@@ -0,0 +1,19 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: "kube-prometheus-prometheus-ingress"
```

Comment:
Could you add labels matching the Helm chart?

```yaml
app: prometheus
release: kube-prometheus
```
Or wouldn't that make sense?

Reply:
For the demo ?

Reply:
Not necessary for the demo, but since you're on this anyway, why not add it now. Easy to forget later.